system-variables.md

```diff
@@ -1,129 +1,838 @@
 ---
-title: The System Variables
-summary: Learn how to use the system variables in TiDB.
+title: System Variables
+summary: Use system variables to optimize performance or alter running behavior.
 category: reference
-aliases: ['/docs/dev/system-variables/','/docs/dev/reference/configuration/tidb-server/mysql-variables/']
+aliases: ['/docs/dev/system-variables/','/docs/dev/reference/configuration/tidb-server/mysql-variables/', '/docs/dev/tidb-specific-system-variables/','/docs/dev/reference/configuration/tidb-server/tidb-specific-variables/']
```

Comment:
The aliases are incomplete. I can't open this page: https://docs.pingcap.com/tidb/dev/tidb-specific-system-variables.
I'll fix it in another PR.

Reply:
I fixed it in https://github.com/pingcap/docs/pull/3239/commits/1c5608e2117f48f0c2ba81ed7470948ecc9997cb. But it is not merged yet. 😂 But thanks anyway, sharp eyes!